src/Request/SiteRequestContext.php

```diff
@@ -31,10 +31,7 @@ class SiteRequestContext extends RequestContext implements SiteRequestContextInt
      */
     protected $selector;
 
-    /**
-     * @var SiteInterface
-     */
-    private $site;
+    private ?SiteInterface $site = null;
```

Comment:
is it nullable?

Reply:
It's not initiliazed.

```
final public function setSite(SiteInterface $site)
    {
        $this->site = $site;
    }

    final public function getSite()
    {
        if (!$this->site instanceof SiteInterface) {
            $this->site = $this->selector->retrieve();
        }

        return $this->site;
    }
```
Coming from a setter.